run.py

```diff
@@ -1,6 +1,12 @@
 """ run.py """
 import os
 from app import create_app
+from os.path import join, dirname
+from dotenv import load_dotenv
+
+# load .env in the base root
+dotenv_path = join(dirname(__file__), '.env')
+load_dotenv(dotenv_path)
 
 config_name = "development"
```

Comment:
Your environment should be defined in your .env

Reply:
Also on line 16 in run.py is there any reason for specifying the port number?

Reply:
Consider this taken care of.